src/http/api/resources/swarm.js

```diff
@@ -10,13 +10,22 @@ exports = module.exports
 // common pre request handler that parses the args and returns `addr` which is assigned to `request.pre.args`
 exports.parseAddrs = (request, reply) => {
   if (!request.query.arg) {
-    return reply("Argument 'addr' is required").code(400).takeover()
+    const err = 'Argument \'addr\' is required'
```

Comment:
can we use '``' instead? `Argument 'addr' is required` - should work without needing to escape anything.

Reply:
Sure!

Reply:
Done @dryajov